src/main/java/org/mastodon/mamut/feature/branch/BranchNSpotsFeature.java

```diff
@@ -46,13 +46,138 @@
 import org.mastodon.properties.IntPropertyMap;
 import org.scijava.plugin.Plugin;
 
+
+
+/**
+ * Represents the number of spots within a branch spot. In many cases this may
+ * be equivalent to {@link BranchDisplacementDurationFeature} (duration).
+ * However, in situations where there are spots missing within a branch spot, this
+ * may well be different.
```

Comment:
It's also different for root nodes. Branch duration for root node is smaller by one that number of spots.